H3Lib/Extensions/DirectionExtensions.cs

```diff
@@ -15,15 +15,15 @@ public static class DirectionExtensions
         /// -->
         internal static Direction Rotate60CounterClockwise(this Direction digit)
         {
-            return digit switch
+            switch (digit)
             {
-                Direction.K_AXES_DIGIT => Direction.IK_AXES_DIGIT,
-                Direction.IK_AXES_DIGIT => Direction.I_AXES_DIGIT,
-                Direction.I_AXES_DIGIT => Direction.IJ_AXES_DIGIT,
-                Direction.IJ_AXES_DIGIT => Direction.J_AXES_DIGIT,
-                Direction.J_AXES_DIGIT => Direction.JK_AXES_DIGIT,
-                Direction.JK_AXES_DIGIT => Direction.K_AXES_DIGIT,
-                _ => digit
+                case Direction.K_AXES_DIGIT: return Direction.IK_AXES_DIGIT;
```

Comment:
I guess I'm not seeing the benefit of restructuring this switch to have returns on the case conditions being met.

I'm not inherently adverse to multiple exit points in a function, but I'm not seeing the advantage here in going back to switch() testing with this syntax.